alibi_detect/utils/visualize.py

```diff
@@ -77,6 +78,11 @@ def plot_feature_outlier_image(od_preds: Dict,
         instance_ids = list(range(len(od_preds['data']['is_outlier'])))
     n_instances = min(max_instances, len(instance_ids))
     instance_ids = instance_ids[:n_instances]
+
+    if outliers_only and n_instances == 0:
+        warnings.warn('No outliers found!', UserWarning, stacklevel=3)
```

Comment:
@mauicv  is the warning swallowed without `stacklevel=3`?

Reply:
Nope, I've removed it!